Python/Extract_Contents_of_Zip/extractScript.py

```diff
@@ -0,0 +1,14 @@
+#import pyshorteners library
```

Comment:
Can you explain to me the use of this file here? @Sapna2001

Reply:
I extracted this script from a zip file to show an example. I will remove it.